### PR TITLE
DIO-2460 - Implement magento modules into redux store

### PR DESCRIPTION
**Pull request by aronmartin**

### CODE REVIEW

File: src/organisms/product/dorel-section-product-features.html [hunk 1]
```diff
@@ -96,106 +90,122 @@
...
+          },
+          /**
+           * @attribute
+           * @name featuresData
+           * @description coming from the page-builder this data object is filled
+           * with ACF/Wordpress data about the component
+           */
+          featuresData: {
+            type: Object,
+            statePath: function(state) {
```
Comment (mrpharderwijk) [inline src/organisms/product/dorel-section-product-features.html]: `state => {...}`

File: src/organisms/product/dorel-section-product-topics.html [hunk 2]
```diff
@@ -1,71 +1,68 @@
...
-          computed: '_cardData(topicData)'
-        },
+          /**
+           * @attribute
+           * @name topicData
+           * @description coming from the redux modules state
+           */
+          topicData: {
+            type: Object,
+            statePath: function(state) {
```
Comment (mrpharderwijk) [inline src/organisms/product/dorel-section-product-topics.html]: `state => {...}`

File: src/organisms/product/dorel-section-product-usps.html [hunk 3]
```diff
@@ -97,36 +89,55 @@ <h3 class="usp-title">[[ usp.title ]]</h3>
...
+
+          /**
+           * @attribute
+           * @name uspsData
+           * @description coming from the page-builder this data object is filled
+           * with ACF/Wordpress data about the component
+           */
+          uspsData: {
+            type: Object,
+            statePath: function(state) {
```
Comment (mrpharderwijk) [inline src/organisms/product/dorel-section-product-usps.html]: `state => {...}`

File: src/organisms/product/dorel-section-product-topics.html (hunk 2 above)
Comment (aronmartin) [reply src/organisms/product/dorel-section-product-topics.html]: idem

File: src/organisms/product/dorel-section-product-usps.html (hunk 3 above)
Comment (aronmartin) [reply src/organisms/product/dorel-section-product-usps.html]: idem

**Pull request merged**